staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -144,7 +144,8 @@ func NewDeltaFIFOWithOptions(opts DeltaFIFOOptions) *DeltaFIFO {
 //
 // DeltaFIFO's Pop(), Get(), and GetByKey() methods return
 // interface{} to satisfy the Store/Queue interfaces, but they
-// will always return an object of type Deltas.
+// will always return an object of type Deltas. List() returns
+// the newest objects currently in the FIFO.
```

Comment:
"the newest objects currently in the FIFO" is kind of vague.  More precise wording might be something like "the newest object from each accumulator" or "... each accumulator in the FIFO".

Reply:
Agreed that's better wording.